sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_policies.py

```diff
@@ -0,0 +1,18 @@
+# coding=utf-8
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+
+from azure.core.pipeline.policies import SansIOHTTPPolicy
+
+
+class CognitiveServicesCredentialPolicy(SansIOHTTPPolicy):
```

Comment:
related to https://github.com/Azure/azure-sdk-for-python/pull/10509 ?

Reply:
#10540